internal/services/notificationhub/notification_hub_namespace_data_source.go

```diff
@@ -99,6 +104,13 @@ func resourceArmDataSourceNotificationHubNamespaceRead(d *pluginsdk.ResourceData
 		if props := model.Properties; props != nil {
 			d.Set("enabled", props.Enabled)
 			d.Set("namespace_type", string(pointer.From(props.NamespaceType)))
+
+			if zoneRedundancy := props.ZoneRedundancy; zoneRedundancy != nil {
+				d.Set("zone_redundant", *zoneRedundancy == namespaces.ZoneRedundancyPreferenceEnabled)
+			} else {
+				d.Set("zone_redundant", false)
+			}
```

Comment:
We can use `pointer.From` and simplify this to:

```suggestion
			d.Set("namespace_type", string(pointer.From(props.NamespaceType)))
			d.Set("zone_redundancy_enabled", pointer.From(props.ZoneRedundancy) == namespaces.ZoneRedundancyPreferenceEnabled)
```